systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1382,7 +1382,10 @@ void testConsumerOffsetFiles() {
         String result = cmdKubeClient().execInPod(KafkaResources.kafkaPodName(CLUSTER_NAME, 0),
                 "/bin/bash", "-c", commandToGetFiles).out();
 
-        assertThat("Folder kafka-log0 has data in files", result.equals(""));
+        // TODO / FIXME
+        //assertThat("Folder kafka-log0 has data in files:\n" + result, result.equals(""));
```

Comment:
This was empty before, as ZKTS didn't use Kafka.
With KSTS, this is already populated -- I guess it can be removed, unless someone has a better idea?